packages/ag-charts-community/src/util/listeners.ts

```diff
@@ -42,6 +66,8 @@ export class Listeners<EventType extends string, EventHandler extends Handler> {
             } catch (e) {
                 Logger.errorOnce(e);
             }
+
+            if (params.length === 1 && immediatePropagationStopped(params[0])) return;
```

Comment:
Could you please clarify which specific handlers you are trying to stop calling? Intuitively, it might make more sense to just check the `defaultPrevented` property in the handlers that are causing problems.